lxc/auth.go

```diff
@@ -1074,6 +1077,50 @@ func (c *cmdIdentityEdit) run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+type cmdIdentityDelete struct {
+	global *cmdGlobal
+}
+
+func (c *cmdIdentityDelete) command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = usage("delete", i18n.G("[<remote>:]<authentication_method>/<name_or_identifier>"))
```

Comment:
Since an identifier here can be (if I understood correctly) a certificate fingerprint for TLS or an email for OIDC, I think some examples or a more detailed description would be helpul here.

Reply:
I've added some examples :)